src/components/List/SimpleList/SimpleList.jsx

```diff
@@ -81,16 +102,16 @@ const SimpleList = ({
       break;
   }
 
-  const [selectedIds, setSelectedIds] = useState([]);
-  const [selectedId, setSelectedId] = useState(null);
-
-  const handleSelect = id => {
-    setSelectedId(selectedIds.indexOf(id) !== -1 ? null : id);
-    setSelectedIds(
-      selectedIds.indexOf(id) !== -1
-        ? selectedIds.filter(item => item !== id)
-        : [...selectedIds, id]
-    );
+  const handleSelect = (id, parentId = null) => {
```

Comment:
I think that `parentId` can be undefined and passed along to `handleEditModeSelect`, but with this default assignment it gives the impression that it must have a value or be null. Is there a purpose of the null assignment that I'm not understanding?